content/html/glossaries/html-tag-a.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: "<a>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Anchor"
+  - "Link"
+  - "URL"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+## Definition 
+
+The HTML tag `<a>`, also known as the _anchor tag_, creates a link to another page or to a location in the current page. It can also be used to link to email addresses, phone numbers, and files.
```

Comment:
Because the glossary page will use the Definition field in the table (see [inspiration here](https://www.w3schools.com/tags/default.asp)), we want the Definition to be as short and concise as possible. So maybe something like:

## Definition

The anchor tag creates a link to another page or to a location on the current page. It can also be used to link to email addresses, phone numbers, and files.

---

Also, I'm thinking no italicize for technical keywords on Codecademy Docs (which is what we usually do for lessons/projects) just because every page on Codecademy Docs will have a lot of keywords, so it might be distracting. What do you think?

Reply:
Done, and I think it's fair to have fewer italics strewn about.